_posts/2023-07-12-future-career-growth-for-frontend-engineer.md

```diff
@@ -0,0 +1,180 @@
+
+P.S Also Google, Amazon and Microsoft provide certifications of various levels for developers, you may want take one.
+
+**Example from the real life:**
+
+In my thrilling journey as a developer 😅, I embarked on a project that involved creating a Telegram bot for a company specializing in selling clothes and shoes. Building a seamless and responsive frontend application for bot using Next.js was our top priority, and we needed a powerful deployment solution that could handle the demands of our fast-growing user base. AWS Amplify emerged as the perfect match for our requirements, offering a seamless deployment experience and a wide range of benefits.
+
+With AWS Amplify, deploying our Next.js-powered Telegram bot became a breeze. The Amplify Console provided a user-friendly interface, allowing us to connect our code repository and set up continuous deployment with just a few clicks. Every time we pushed changes to our Git repository, Amplify automatically triggered the build and deployment process, ensuring that our users always had access to the latest features and improvements.
+
+One of the standout features of AWS Amplify was its ability to handle serverless functions effortlessly. We leveraged this capability to integrate the bot with various third-party services, enabling smooth payment processing and real-time inventory updates. Amplify's serverless architecture was not only cost-effective but also highly scalable, ensuring that our Telegram bot could accommodate any surge in user activity during flash sales or promotional events.
+
+Without knowledge of AWS services, for development and deployment of the new application for bot could be challenging and I most likely it would be necessary to look for a person who can deploy the frontend on the cloud provider.
+
+**Useful materials (Yes, all links bound to AWS, because I prepared to AWS developer associate certification and passed it):**
+
+- [https://www.udemy.com/course/aws-certified-developer-associate-dva-c01/](https://www.udemy.com/course/aws-certified-developer-associate-dva-c01/)
+- [https://www.udemy.com/course/aws-certified-developer-associate-practice-tests-dva-c01/](https://www.udemy.com/course/aws-certified-developer-associate-practice-tests-dva-c01/)
+- [https://www.udemy.com/course/aws-developer-associate-practice-exams/](https://www.udemy.com/course/aws-developer-associate-practice-exams/)
+- [https://aws.amazon.com/getting-started/hands-on/build-react-app-amplify-graphql/](https://aws.amazon.com/getting-started/hands-on/build-react-app-amplify-graphql/)
+
+**Real examples**
+- [Angular Application Deployment to Azure Web App using GitHub](https://blog.devgenius.io/angular-application-deployment-to-azure-web-app-using-github-49a4e3a7ba87)
```

Comment:
I guess you can point to one of our own topics - https://engineering.backbase.com/2022/06/08/deploying-angular-app-on-azure :)